pallets/manta-sbt/src/lib.rs

```diff
@@ -613,6 +613,25 @@ pub mod pallet {
             Self::deposit_event(Event::<T>::ChangeFreeReserveAccount { account });
             Ok(())
         }
+
+        /// Remove allowlist evm account. Requires `AdminOrigin`
+        #[pallet::call_index(8)]
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::change_allowlist_account())]
+        #[transactional]
+        pub fn remove_allowlist_evm_account(
```

Comment:
this is unneeded you can just use `None` in `change_allowlist_account`

Reply:
`change_allowlist_account` is used to set privilege account. while this `remove_allowlist_evm_account` used to remove user eth address storage, normally this is only used on test environment.

Reply:
Ah gotcha I misread this, is fine